src/app/views/provisioning/subscriptions.html

```diff
@@ -1,6 +1,11 @@
-<h2 class="section-title">{{ 'mno_enterprise.templates.dashboard.provisioning.subscriptions.title' | translate }}</h2>
+<h2 class="section-title" translate>{{ vm.subscriptionsTitle() }}</h2>
```

Comment:
Instead of having the method vm.subscriptionsTitle, I would do:
```html
<h2 class="section-title" ng-show="vm.cartSubscriptions" translate>'mno_enterprise.templates.dashboard.provisioning.subscriptions.cart.title'</h2>
<h2 class="section-title" ng-hide="vm.cartSubscriptions" translate>'mno_enterprise.templates.dashboard.provisioning.subscriptions.title'</h2>
```